automation/deploy_pgcluster.yml

```diff
@@ -400,6 +400,9 @@
 
     - role: netdata
 
+    - role: prometheus
+      when: with_metric_exporters | bool
```

Comment:
perhaps it is better to rename with_metric_exporters to `prometheus_metrics` or `prometheus_exporters`.